Description: Skip tests failing in test environment
 These tests generally work but fail in a Ubuntu autopkgtest environment.
 This is consistent across all architectures.
 Skip those as they do not represent real regressions.
Forwarded: no
X-Not-Forwarded-Reason: This only applies to autopkgtest environments
Author: Christian Ehrhardt <christian.ehrhardt@canonical.com>
Last-Update: 2021-07-13
--- a/app/test/meson.build
+++ b/app/test/meson.build
@@ -220,7 +220,7 @@
         ['fib6_autotest', true],
         ['func_reentrancy_autotest', false],
         ['flow_classify_autotest', false],
-        ['hash_autotest', true],
+        ['hash_autotest', false],
         ['interrupt_autotest', true],
         ['ipfrag_autotest', false],
         ['lcores_autotest', true],
@@ -236,10 +236,10 @@
         ['memzone_autotest', false],
         ['meter_autotest', true],
         ['multiprocess_autotest', false],
-        ['per_lcore_autotest', true],
+        ['per_lcore_autotest', false],
         ['pflock_autotest', true],
         ['prefetch_autotest', true],
-        ['rcu_qsbr_autotest', true],
+        ['rcu_qsbr_autotest', false],
         ['red_autotest', true],
         ['rib_autotest', true],
         ['rib6_autotest', true],
@@ -254,7 +254,7 @@
         ['stack_autotest', false],
         ['stack_lf_autotest', false],
         ['string_autotest', true],
-        ['table_autotest', true],
+        ['table_autotest', false],
         ['tailq_autotest', true],
         ['ticketlock_autotest', true],
         ['timer_autotest', false],
@@ -274,7 +274,7 @@
         ['power_autotest', true],
         ['power_kvm_vm_autotest', false],
         ['reorder_autotest', true],
-        ['service_autotest', true],
+        ['service_autotest', false],
         ['thash_autotest', true],
         ['trace_autotest', true],
 ]
